pull.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import plotly.offline as py
+import plotly.graph_objs as go
+
+index = ['MU','GE','AMD','BAC']
+
+
+def main():
+    stocks = pd.DataFrame()
+    i = 0
+    while i < len(index):
+        r = requests.get('https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol=' + str(
```

Comment:
String concat like this is expensive. Look into using the str.format() function. 

You should look into using an enhanced for loop. Check this out:
for ticker in index:
   r = requests.get('https://www.alphavantage.co/query?function=TIME_SERIES_DAILY&symbol={}'&datatype=csv&apikey=603MDIJGG9TGNV60;.format(ticker)

You can then get rid of the i increment variable and the while loop.